api/envoy/api/v2/rds.proto

```diff
@@ -33,8 +33,7 @@ service RouteDiscoveryService {
   rpc StreamRoutes(stream DiscoveryRequest) returns (stream DiscoveryResponse) {
   }
 
-  rpc IncrementalRoutes(stream IncrementalDiscoveryRequest)
-      returns (stream IncrementalDiscoveryResponse) {
+  rpc DeltaRoutes(stream DeltaDiscoveryRequest) returns (stream DeltaDiscoveryResponse) {
```

Comment:
This method name change is not wire compatible (it changes the URL), thus is against [Breaking change policy](https://github.com/envoyproxy/envoy/blob/master/CONTRIBUTING.md#breaking-change-policy).

Reply:
I'd have to hear from a more experienced Envoy person to be sure, but I think this one is an exception because we are sure that both sides of this gRPC service are completely unimplemented until now.

Reply:
Yes, incremental xDS is entirely experimental, so this is legit.

Reply:
Is this indicated in the proto file somewhere, or is there some other way we should be able to know what is experimental and what is considered "frozen"?

Reply:
In this specific case, it's well known in the Envoy/xDS community that this feature is highly experimental; there is no client or server in the wild that implement this as it's a moving target. You can track this work at https://github.com/envoyproxy/envoy/issues/4991.